src/CSET/__init__.py

```diff
@@ -133,7 +134,7 @@ def main():
     )
     parser_recipe_id.set_defaults(func=_recipe_id_command)
 
-    cli_args = sys.argv[1:] + os.getenv("CSET_ADDOPTS", "").split()
+    cli_args = sys.argv[1:] + shlex.split(os.getenv("CSET_ADDOPTS", ""))
```

Comment:
I eventually remembered that python has a shell style parser in its standard library: https://docs.python.org/3/library/shlex.html